src/main/java/Objects/Packets/Botnet.java

```diff
@@ -2,10 +2,11 @@
 
 public class Botnet extends Packet {
 
-    public Botnet(final int team) {
-        //super(team);
-        /*setCost(1);
-        setStealth(0);*/
+    public Botnet(final String team) {
+        setPacketType("Botnet");
+        setTeam(team);
+        setCost(1);
+        setStealth(0);
     }
 
```

Comment:
this is chaos

Reply:
So is walking around with those freakish weights on your hands.